### PR TITLE
Create rbo.yml

### PR DESCRIPTION
**Pull request by DanBerrios**
Submitted request for inclusion in OBO Foundry, following ODK instructions for metadata and purl configs.

### CODE REVIEW

**Comment by jamesaoverton** [comment]
Waiting on https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1348

File: config/rbo.yml
Comment (jamesaoverton) [inline config/rbo.yml:7]: Note that these two `products` will not resolve as specified here, because there are no `rbo.owl` or `rbo.obo` files at the top level of your repository.

Comment (jamesaoverton) [reply config/rbo.yml:7]: Related to https://github.com/DanBerrios/RBO/issues/64

Comment (DanBerrios) [reply config/rbo.yml:7]: Thanks James!  I had to config my local git ignorecase to be false. I've done this and now I have added rbo.owl the GH repo.  Anything else needed?

Comment (jamesaoverton) [reply config/rbo.yml:7]: Thanks! This is fixed, and everything else looks good.

**Review by jamesaoverton — APPROVED** [review]

**Comment by jamesaoverton** [comment]
This is ready to go, and I'll merge it after RBO is approved and an `rbo.md` is added to the registry.

**Comment by matentzn** [comment]
see https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1423

**Review by DanBerrios — COMMENTED** [review]
Looks correct.

**Pull request merged**
